src/content/get-started/install/openshift.mdx

```diff
@@ -0,0 +1,192 @@
+- `<cluster-name>`: for example, `okteto`
+
+## Getting your Okteto License
+
+A license is mandatory to use Okteto. You'll receive a license key as part of your subscription to Okteto. If you haven't received it, [please open a support ticket](https://okteto.com/support).
+
+If you are interested in evaluating Okteto, [sign up for our free 30 days trial](https://www.okteto.com/free-trial/). No credit card required.
+
+## Steps before Installing Okteto
+
+Before installing Okteto, you will need to follow these steps:
+
+- Create the `okteto` namespace
+- Allow containers as `root` in the `okteto` namespace
+- Allow Okteto Build service privileged access
+- Allow Okteto Daemon service to access the host
+- Copy Openshift wildcard certificate to the `okteto` namespace
+- Get the OpenShift Ingress Controller IP
+
+### Create the `okteto` namespace
+
+Run the following command to create the `okteto` namespace:
+
+```bash
+kubectl create namespace okteto
+```
+
+### Allow containers as `root` in the `okteto` namespace
+
+By default, running `root` containers in OpenShift is not allowed. Today, most of the Okteto Control Plane runs as `root` (this is something we will fix soon). To allow `root` containers for the Okteto Control Plane components, run the following commands:
+
+```bash
+oc adm policy add-scc-to-user anyuid -z default -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto-buildkit -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto-registry -n okteto 
+```
+
+### Allow Okteto Build service privileged access
+
+By default, running privileged containers in OpenShift is not allowed. This restriction impacts the installation of the Okteto Build. To give privileged access to the Okteto Build service, run the following command:
+
+```bash
+ oc adm policy add-scc-to-user privileged -z okteto-buildkit -n okteto
+```
+
+### Allow the Okteto Daemon service to access the host
+
+By default, access to the host in OpenShift is not allowed. This restriction impacts the installation of the Okteto Daemon service. To give access to the host to the Okteto Daemon run the following command:
+
+```bash
+oc adm policy add-scc-to-user hostaccess -z okteto -n okteto
+```
+
+### Copy OpenShift wildcard certificate to the `okteto` namespace
+
+OpenShift provides the wildcard certificate for the wildcard domain `apps.<your-openshift-subdomain>` . We recommend reusing it when installing Okteto.
+
+To do that, copy the certificate to the `okteto` namespace (replace `<cluster-name>` with your cluster name):
+
+```yaml
+kubectl get secret <clustername>-primary-cert-bundle-secret -n openshift-ingress -o yaml | sed 's/namespace: openshift-ingress/namespace: okteto/' | kubectl apply -f -
+```
+
+### Get the OpenShift Ingress Controller IP
+
+We recommend installing Okteto behind the Openshift Ingress controller. To do this, remember the IP of your Openshift Ingress Controller:
```

Comment:
why do we recommend this? are there other viable options?

Reply:
In this kind of SH scenarios, asking for our own LD is hard. We recommend this setup for the initial installation